### PR TITLE
Fixes package license reference and adds Office icon

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: DocumentFormat.OpenXml.Package.props [hunk 1]
```diff
@@ -17,6 +17,10 @@
     <PackageTags>openxml office</PackageTags>
     <PackageReleaseNotes><![CDATA[
```
Comment (twsouthwick) [inline DocumentFormat.OpenXml.Package.props]: nit: extra newline

File: README.md [hunk 2]
```diff
@@ -130,6 +130,13 @@ Known Issues
 Change Log
 -----------
 
+**Version 2.8.1** : January 3, 2018
+
+- Corrected package license file reference.
+- Added Office icon to Nuget package properties.
```
Comment (twsouthwick) [inline README.md]: Should there be a colon after this, since the icon is included?

File: README.md [hunk 3]
```diff
@@ -165,7 +172,7 @@ Change Log
 Documentation
 -------------
 
-The functionality of the specific classes in Open XML SDK Version 2.8.0 is similar to version 2.5, therefore the [Open XML SDK 2.5 for Office](http://msdn.microsoft.com/en-us/library/office/bb448854.aspx) documentation available on MSDN is still accurate.
+The functionality of the specific classes in Open XML SDK Version 2.8.1 is similar to version 2.5, therefore the [Open XML SDK 2.5 for Office](http://msdn.microsoft.com/en-us/library/office/bb448854.aspx) documentation available on MSDN is still accurate.
```
Comment (twsouthwick) [inline README.md]: It may be good to remove the reference to the latest version so we don't need to keep updating it

Comment (msft-ray) [reply README.md]: Good idea. I suggest we change "... in Open XML SDK version 2.8.1 is..." to "... in this version of the Open XML SDK is...".

Comment (tomjebo) [reply README.md]: I'll update

File: DocumentFormat.OpenXml.Package.props [hunk 4]
```diff
@@ -17,6 +17,10 @@
...
 
+Version 2.8.1 : January 3, 2018
+- Corrected package license file reference.
+- Added Office icon for Nuget package.
+
 Version 2.8.0 : December 28, 2017
 - Added default runtime directive for better .NET Native support
```
Comment (twsouthwick) [inline DocumentFormat.OpenXml.Package.props]: There are periods at the end of all of these in the README - I don't think @msft-ray's change updated this props file

Comment (tomjebo) [reply DocumentFormat.OpenXml.Package.props]: @twsouthwick sorry, I meant to add periods as well.

**Pull request merged**